app/scripts/components/common/page-header/logo-container.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ComponentType } from 'react';
+import { Tip } from '../tip';
+import { LinkProperties } from '../card';
+import { Brand, PageTitleSecLink } from './logo';
+/**
+ * LogoContainer that is meant to integrate in the default page header without the dependencies of the veda virtual modules
+ * and expects the Logo SVG to be passed in as a prop - this will support the instance for refactor
+ */
+
+export default function LogoContainer ({ linkProperties, Logo, title, subTitle, version }: {
+  linkProperties: LinkProperties,
+  Logo: JSX.Element,
+  title: string,
+  subTitle: string,
+  version: string
+}) {
+  const LinkElement: ComponentType<any> = linkProperties.LinkElement as ComponentType<any>;
+
+  return (
+    <Brand>
+      <LinkElement {...{[linkProperties.pathAttributeKeyName]: '/'}}>
+        {Logo}
+        <span>{title}</span> <span>{subTitle}</span>
+      </LinkElement>
+      <Tip content={`v${version}`}>
+      <PageTitleSecLink {...{as: linkProperties.LinkElement as ComponentType<any>, [linkProperties.pathAttributeKeyName]: '/development'}}>Beta</PageTitleSecLink>
```

Comment:
I am  not sure if this component can be used by other instances when only logo svg is tweak-able - (ex. ghg overwrote the whole logo)